core/src/main/java/org/arquillian/smart/testing/configuration/Configuration.java

```diff
@@ -114,6 +118,14 @@ public void setAutocorrect(boolean autocorrect) {
         this.autocorrect = autocorrect;
     }
 
+    public void setCustomStrategies(String[] customStrategies) {
+        this.customStrategies = customStrategies;
+    }
+
+    public String[] getCustomStrategies() {
+        return customStrategies;
+    }
+
     public List<ConfigurationItem> registerConfigurationItems() {
         List<ConfigurationItem> configItems = new ArrayList<>();
         configItems.add(new ConfigurationItem("strategies", SMART_TESTING, new String[0]));
```

Comment:
Because @MatousJobanek mentioned in one previous review (and I think he was right) that if system properties are read in `Configuration` class, before this PR the system properties of custom strategies were read in https://github.com/arquillian/smart-testing/blob/master/mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/dependencies/StrategyDependencyResolver.java#L45 then some sys props were read from `Configuration` and others from `StrategyDependencyResolver`, to put all in same place you need to configure each property that is going to be read in `Configuration` class in `configItems`

Reply:
Wouldn't it be great to have only one simple way though?

Reply:
there is only one way, and this is by registering the attribute you want to read either from configuration file or system property. @dipak-pawar can give you more details, but in essence this class just read a property from file, then check if there is a system property overriding it and if not, if a default value is provided, then this value is the one set in `Configuration` class.